terraform/byo-vpc/byo-db/byo-ecs/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "fleet_config" {
   type = object({
     mem                          = optional(number, 4096)
     cpu                          = optional(number, 512)
-    image                        = optional(string, "fleetdm/fleet:v4.50.2")
+    image                        = optional(string, "fleetdm/fleet:v4.51.0")
```

Comment:
The default is actually specified again on line 100 and hasn't been updated.  This is true for the other modules as well.

Reply:
@rfairburn Thanks for catching. I wonder why the script didn't replace them... will fix manually.

Reply:
I'd just noticed this when extending the module for sidecars.  It's the same in the root module and all of the byo-* modules.

Reply:
@rfairburn Should all be fixed. There were several that were still on 4.31.1 for some reason, so they weren't getting picked up on the search and replace in the script.